doc/source/whatsnew/v0.18.0.txt

```diff
@@ -248,6 +248,35 @@ Subtraction by ``Timedelta`` in a ``Series`` by a ``Timestamp`` works (:issue:`1
    ser
    pd.Timestamp('2012-01-01') - ser
 
+
+Signature change for Series.rank and DataFrame.rank and their GroupBy equivalents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The **new** signature is ``.rank(method='average', axis=0, numeric_only=None, na_option='keep', ascending=True, pct=False)``.
+
```

Comment:
hmm, this is what I did for sorting, so precedent for what you have here as well.

```
In [1]: Series.sort_values?
Type:        instancemethod
String form: <unbound method Series.sort_values>
File:        c:\miniconda\envs\neat\lib\site-packages\pandas\core\series.py
Definition:  Series.sort_values(self, axis=0, ascending=True, inplace=False, kind='quicksort', na_position='last')
Docstring:
Sort by the values along either axis

.. versionadded:: 0.17.0

Parameters
----------
by : string name or list of names which refer to the axis items
axis : index to direct sorting
ascending : bool or list of bool
     Sort ascending vs. descending. Specify list for multiple sort orders.
     If this is a list of bools, must match the length of the by
inplace : bool
     if True, perform operation in-place
kind : {`quicksort`, `mergesort`, `heapsort`}
     Choice of sorting algorithm. See also ndarray.np.sort for more information.
     `mergesort` is the only stable algorithm. For DataFrames, this option is
     only applied when sorting on a single column or label.
na_position : {'first', 'last'}
     `first` puts NaNs at the beginning, `last` puts NaNs at the end

Returns
-------
sorted_obj : Series
```